Payload json to tsvs@0.1.0

edsu7 opened the pull request.

edsu7 committed:
[wfpm v0.8.0] added starting template for payload-json-to-tsvs@0.1.0

edsu7 committed:
initial commit

edsu7 committed:
Update correct `readme`

- undid previous update to wrong `argo-data-submission-wf/readme.md`
- updated `payload-json-to-tsvs/readme.md`

lindaxiang commented on payload-json-to-tsvs/README.md:
````diff
@@ -0,0 +1,49 @@
+params file(s) can be found in the `tests` folder.
+
+```
+nextflow run icgc-argo/argo-data-submission/download-aspera/main.nf -r download-aspera.v0.1.0 -params-file <your-params-json-file>
````
download-aspera??

edsu7 replied:
Fixed:
https://github.com/icgc-argo/argo-data-submission/blob/e39c1728161f2b32327f9885b78a638bc1304c28/payload-json-to-tsvs/README.md?plain=1#L39

lindaxiang commented on payload-json-to-tsvs/main.py:
```diff
@@ -0,0 +1,172 @@
+    make_experiment_tsv(metadata,args.data_directory)
+    make_file_tsv(metadata,args.data_directory)
+    make_read_group_tsv(metadata,args.data_directory)
+
```
`data_directory` is only needed by `make_file_tsv`.

edsu7 replied:
Fixed.

edsu7 committed:
updates to `main.py` and `readme.md`

- removed unused `data_directory` argument in some functions in `main.py`
- corrected typos in `readme.md`
